hudi-common/src/test/java/org/apache/hudi/common/util/TestFSUtils.java

```diff
@@ -43,11 +49,33 @@
 import java.util.stream.Stream;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 /**
  * Tests file system utils.
  */
 public class TestFSUtils extends HoodieCommonTestHarness {
+  private long minRollbackToKeep = 10;
```

Comment:
We shouldn't have counter variables as members of a test class, each test might mutate it and eventually break other tests

Reply:
incorporated the comments.